### PR TITLE
Feature mapping

### PR DESCRIPTION
**Pull request by Monjoie11**

### CODE REVIEW

File: com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java [hunk 1]
```diff
@@ -1,64 +1,154 @@
...
 	}
-	public void setAffiliatedCompanyName(String affiliatedCompanyName) {
-		this.affiliatedCompanyName = affiliatedCompanyName;
+
+	public void setAffiliatedCompany(Company affiliatedCompany) {
+		this.affiliatedCompany = affiliatedCompany;
 	}
-	public Object getTimeCreated() {
+
+	public LocalDateTime getTimeCreated() {
```
Comment (Shalom-Nguyen) [inline com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java]: PostTime or timeOfPost | choose one and remain consistency

File: com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java [hunk 2]
```diff
@@ -1,64 +1,154 @@
...
 		return timeCreated;
 	}
-	public void setTimeCreated(Object timeCreated) {
+
+	public void setTimeCreated(LocalDateTime timeCreated) {
```
Comment (Shalom-Nguyen) [inline com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java]: PostTime or timeOfPost

File: com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java [hunk 3]
```diff
@@ -1,64 +1,154 @@
...
 		this.timeCreated = timeCreated;
 	}
-	public Object getStartTime() {
+
+	public LocalDateTime getStartTime() {
```
Comment (Shalom-Nguyen) [inline com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java]: ShiftStartTime

File: com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java [hunk 4]
```diff
@@ -1,64 +1,154 @@
...
 		return startTime;
 	}
-	public void setStartTime(Object startTime) {
+
+	public void setStartTime(LocalDateTime startTime) {
```
Comment (Shalom-Nguyen) [inline com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java]: ShiftStartTime

File: com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java [hunk 5]
```diff
@@ -1,64 +1,154 @@
...
 		this.startTime = startTime;
 	}
-	public Object getEndTime() {
+
+	public LocalDateTime getEndTime() {
```
Comment (Shalom-Nguyen) [inline com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java]: ShiftEndTime

File: com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java [hunk 6]
```diff
@@ -1,64 +1,154 @@
...
 		return endTime;
 	}
-	public void setEndTime(Object endTime) {
+
+	public void setEndTime(LocalDateTime endTime) {
```
Comment (Shalom-Nguyen) [inline com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java]: ShiftEndTime

File: com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java [hunk 7]
```diff
@@ -1,64 +1,154 @@
...
+		this.endTime = endTime;
+	}
+
+	public Post(int postId, String content, User postingUser, Set<User> acceptingUser, Company affiliatedCompany,
```
Comment (Shalom-Nguyen) [inline com.revature.proj2/src/main/java/com/revature/project2/pojos/Post.java]: ShiftStartTime/EndTime --- postContent --- postTime | make appropriate changes to the auto generated code

File: com.revature.proj2/src/main/java/com/revature/project2/pojos/User.java [hunk 8]
```diff
@@ -1,23 +1,69 @@
...
+	@Column(name="EMAIL")
 	private String email;
+	
+	@Column(name="PASSWORD")
 	private String password;
+	
+	@Column(name="FIRST_NAME")
 	private String firstName;
+	
+	@Column(name="LASTNAME")
```
Comment (Shalom-Nguyen) [inline com.revature.proj2/src/main/java/com/revature/project2/pojos/User.java]: LAST_NAME

Comment (Monjoie11) [reply com.revature.proj2/src/main/java/com/revature/project2/pojos/User.java]: change made in javav and sql

**Pull request closed without merging**